src/chains/ethereum/ethereum/src/miner/executables.ts

```diff
@@ -1,7 +1,42 @@
 import { TypedTransaction } from "@ganache/ethereum-transaction";
 import { Heap } from "@ganache/utils";
+import { byNonce } from "../transaction-pool";
 
-export type Executables = {
-  inProgress: Set<TypedTransaction>;
-  pending: Map<string, Heap<TypedTransaction>>;
-};
+export class Executables {
```

Comment:
Just a though - feel free to dismiss: Would this make sense to be named `ExecutableTransactionContainer` or something more explicit?

Reply:
Hmm, yeah, now that this is a class maybe `Executables` isn't the best name. I'm cool with that name, fixing!